deployment/lifelines

```diff
@@ -1,12 +1,14 @@
 $wait Run the l10n.js script to generate sys_L10nString.csv
-$wait If you haven't set the password for the user in this script, kill it now and alter it
+$wait If you haven't set the password for the users in this script, kill it now and alter it
 
 # Add group
 add group lifelines
 
 # Create user
```

Comment:
single user or multiple users? I can't tell

Reply:
'single user or multiple users? I can't tell' what cant you tell , there are 2 users , both have a pw, these need to be set when running the script en removed after completion, as stated in the $wait commands

Reply:
ahh, now i get it, you make an admin user and then an importer user

Reply:
but then it should be # create users as well

Reply:
updated the code comment